comp/metadata/host/hostimpl/status.go

```diff
@@ -43,15 +43,13 @@ func (h *host) populateStatus(stats map[string]interface{}) {
 	json.Unmarshal(hostnameStatsJSON, &hostnameStats) //nolint:errcheck
 	stats["hostnameStats"] = hostnameStats
 
-	ctx := context.Background()
+	payload := utils.GetFromCache(context.TODO(), h.config)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** Why use `context.TODO()` instead of `context.Background()` ?‏

Reply:
https://reintech.io/term/context-background-and-context-todo-in-go

The way I see things is that the context should be pass down from the caller. Unfortunately the status component does not have a way to accept the callers context, or even if it makes sense. Still, I like to use `TODO()` as a placeholder if in the future we want to refactor things around